staging/src/k8s.io/apimachinery/pkg/labels/selector.go

```diff
@@ -217,21 +217,21 @@ func (r *Requirement) Matches(ls Labels) bool {
 		}
 		lsValue, err := strconv.ParseInt(ls.Get(r.key), 10, 64)
 		if err != nil {
-			klog.V(10).Infof("ParseInt failed for value %+v in label %+v, %+v", ls.Get(r.key), ls, err)
+			klog.V(5).Infof("ParseInt failed for value %+v in label %+v, %+v", ls.Get(r.key), ls, err)
 			return false
 		}
 
 		// There should be only one strValue in r.strValues, and can be converted to a integer.
 		if len(r.strValues) != 1 {
-			klog.V(10).Infof("Invalid values count %+v of requirement %#v, for 'Gt', 'Lt' operators, exactly one value is required", len(r.strValues), r)
+			klog.V(5).Infof("Invalid values count %+v of requirement %#v, for 'Gt', 'Lt' operators, exactly one value is required", len(r.strValues), r)
```

Comment:
this and the change above might spam the logs _a lot_. 10 seems reasonable here.

Reply:
Thanks for looking at this.

Actually, the point of the PR is that the Logging Conventions now specify that 5 is the highest level of detail.  So if someone specified a 5 (as they would if the read the documents) then they would never see this message at a 10.  
Additionally, the Logging Conventions say that for most users "the practical log level is a 2", so logging these messages at a 5 should create no additional log spam.
My reading of the code is that these 3 messages are non-critical errors that should not break the operation, but are errors that should be logged as something is technically incorrect.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

Reply:
Agree that flattening all logs above V(4) to a single level without considering the resulting volume is problematic

Reply:
Hi liggitt, thanks for the review comment.
I did look through all the other log messages as part of this and didn't find anything that stood out as not being in line with the Logging Conventions, aside from these 3 lines.  Again, keep in mind that according to the Conventions there should be nothing over 5 so effectively leaving these messages at 10 would mean they "don't exist".  If these messages are indications of a non-fatal parsing error, then I as a user would want a message to let me know.  According to the comments in code these messages should not be happening (and thus should not be adding to 'log spam), but if they are then capturing them in the log is appropriate so the source of the problem can be fixed.